### PR TITLE
Lawrence recurring payout renewal

### PR DESCRIPTION
**Pull request by tomquirk**
Reverts peeldao/proposals#23

### CODE REVIEW

**Review by wraeth-eth — APPROVED** [review]

**Comment by tomquirk** [comment]
✅  I'm in favor of this proposal. It means a payout for this week, next week and the week after.

**Comment by peripheralist** [comment]
Disapproving this proposal. Reasons have been shared in discord but also covering them here:

Lawrence has been researching user interaction with the juicebox.money app, and proposing UI changes based on that research. My main reasons for disapproving are:
1) The UI proposals so far have not shown much exploration beyond rearranging existing content
2) I haven't been able to find any explanation on how the research has informed any UI changes
3) There seems to be some process/documentation in the Miro board that was shared with the Peel team, but since it doesn't have any clear tasks, findings or hypotheses, i'm unsure of how valuable this is to the rest of the team
4) Right now there isn't a clear need for additional UI contributions on the Peel team

**Comment by jmilldotdev** [comment]
Disapproving for reasons similar to @peripheralist. 

tl;dr
- Lack of concrete deliverables up to this point
- Unclear that Peel has enough formal design work that two full-time design-only contributors are necessary. I do not feel designs are our bottleneck.

**Comment by tomquirk** [comment]
Benjamin is on holiday and so we should assume he abstains from this proposal.

Therefore, this proposal hasn't garnered enough approvals to pass.

@Majordalaw thank you very much for your contributions thus far. Your user testing work has been valuable and insightful. I would encourage you to come chat again with Peel in the future in case we need more UX help.

You've been paid for this funding cycle but your payout will removed starting FC32 (next week). Please wrap up as much work as you can. We'd all love to see something tangible produced for the homepage.
